FirebaseCore/Internal/Sources/HeartbeatLogging/HeartbeatsPayload.swift

```diff
@@ -115,8 +115,8 @@ extension HeartbeatsPayload: HTTPHeaderRepresentable {
 // MARK: - Static Defaults
 
 extension HeartbeatsPayload {
-  /// Convenience instance that represents an empty payload.
-  static let emptyPayload = HeartbeatsPayload()
+  /// Conveniently creates an  instance that represents an empty payload.
+  static var emptyPayload: Self { HeartbeatsPayload() }
```

Comment:
> ```swift
> /// Convenience instance that represents an empty payload.
>   static let emptyPayload = HeartbeatsPayload()
> ```
> Gave the following error:
> > error: static property 'emptyPayload' is not concurrency-safe because non-'Sendable' type 'HeartbeatsPayload' may have shared mutable state
>   static let emptyPayload = HeartbeatsPayload()
>              ^

I think this refactor is likely reasonable, just wanted to clarify what the mutable state is in `HeartbeatsPayload`. Looking at the code quickly, I'm not quite sure why `HeartbeatsPayload` (or its nested types `UserAgentPayload` and `CodingKeys`) wouldn't conform to `Sendable`.

Reply:
Good point, thanks Andrew! I will fix.

Reply:
And then should it go back to `let`?

Reply:
Yeah, it might be a little more efficient than generating a new instance.